Python_Scripts/SimAllSettings.py

```diff
@@ -36,7 +38,7 @@ def simulate(atol, rtol, linSol, iter, solAlg, maxStep, study_indicator):
     counter = 0
 
     # set number of repetitions for simulation
-    sim_rep = 40
+    sim_rep = 10
```

Comment:
Is this script used during the trajectory comparison? I hope not...

Reply:
No :laughing: I also thought so first, but it is only used in the main study.

Reply:
Oh, Will add a TODO to increase that number again for the final version.